src/main/kotlin/com/autonomousapps/internal/utils/moshi.kt

```diff
@@ -78,12 +78,8 @@ inline fun <reified K, reified V> String.fromJsonMapList(): Map<K, List<V>> {
   return adapter.fromJson(this)!!
 }
 
-inline fun <reified T> List<T>.toPrettyString(withNulls: Boolean = false): String {
-  return getJsonListAdapter<T>(withNulls).indent("  ").toJson(this)
-}
-
-inline fun <reified T> Set<T>.toPrettyString(withNulls: Boolean = false): String {
-  return getJsonSetAdapter<T>(withNulls).indent("  ").toJson(this)
+inline fun <reified T> T.toPrettyString(withNulls: Boolean = false): String {
```

Comment:
does this actually work? I thought we had to be explicit about it being a `Collection<T>`

Reply:
Let me double checked but yes, I think it did for me.

Reply:
Good find; thanks.

Reply:
I ended up merging this bit onto master separately, so you may have a minor merge conflict here when you rebase.